src/main/java/org/candlepin/subscriptions/marketplace/MarketplaceProperties.java

```diff
@@ -39,6 +39,12 @@
      */
     private String apiKey;
 
+    /**
+     * TEMPORARY setting for returning a dummy subscription ID. Remove once
+     * MarketplaceSubscriptionCollector is fully implemented.
+     */
+    private String dummyId = "DUMMY";
```

Comment:
Can you please alias to the env var `MARKETPLACE_DUMMY_ID` in `src/main/resources/application-marketplace.yaml`?